static/js/components/VideoPlayer_test.js

```diff
@@ -304,4 +303,30 @@ describe("VideoPlayer", () => {
       }
     })
   })
+  ;[false, true].forEach(function(useYoutube) {
+    it(`checkYouTube ${expect(
+      useYoutube
+    )} be called if useYoutube is ${String(useYoutube)}`, async () => {
+      const wrapper = renderPlayer({ useYouTube: useYoutube })
+      const checkStub = sandbox.stub(wrapper.instance(), "checkYouTube")
+      wrapper.instance().componentDidMount()
+      sinon.assert.callCount(checkStub, useYoutube ? 1 : 0)
```

Comment:
this is probably stupid but you could use `Number(useYoutube)` here instead of the ternary :laughing:

Reply:
✅